tests/framework/extensions/users/users.go

```diff
@@ -302,7 +302,7 @@ func RemoveClusterRoleFromUser(rancherClient *rancher.Client, user *management.U
 		Duration: 100 * time.Millisecond,
 		Factor:   1,
 		Jitter:   0,
-		Steps:    5,
+		Steps:    10,
```

Comment:
This test was flaking out often in CI. I figured we can afford to wait an extra 500ms to give it a better chance at passing.